src/builder.rs

```diff
@@ -96,24 +97,71 @@ impl Builder {
                     ..Default::default()
                 })
             }
+            ForkName::Deneb => {
+                let withdrawals = payload_attributes
+                    .withdrawals()
+                    .map_err(|_| Error::LogicError)?
+                    .clone()
+                    .into();
+                ExecutionPayload::Deneb(ExecutionPayloadDeneb {
+                    parent_hash,
+                    timestamp,
+                    fee_recipient,
+                    prev_randao,
+                    block_number,
+                    gas_limit,
+                    transactions,
+                    withdrawals,
+                    ..Default::default()
```

Comment:
This is where the `block_hash` is being set

Reply:
Are you planning to implement unblinding of the payloads @jimmygchen? It'll require a little bit of tweaking, as the unblinding endpoint doesn't exist at all in `dummy_builder` at the moment

Reply:
This defaults to zero hash right?

I [had to calculate the block hash](https://github.com/realbigsean/mock-relay/pull/7/files#diff-5638bfe07e1a7fc6ed9c4a280392f37b80d227aa1ac86459169ddac3fcc24bd9R240-R244) when I was trying to get `mock-relay` to work.

I'd probably leave unblinding out for now unless we need it for `blockprint`. I think we've got sufficient happy path local testing with `dummy-builder` and `mock-relay` for now. 

I'm planning to try out [Kurtosis](https://github.com/kurtosis-tech/eth2-package) next, it looks like the Kurtosis team have done some work on test cases for deneb mev flow.

Reply:
Yeah `Hash256::default() == 0x0`.

Blockprint probably won't use `dummy_builder` any more, so if if this good enough for your testing I'm happy to merge it!

Reply:
I ended up implementing unblind 🙈 and also got `get_header` to return a dummy hash so that CL accepts it.